src/code42cli/sdk_client.py

```diff
@@ -31,13 +32,18 @@ def create_sdk(profile, is_debug_mode, totp=None):
             requests.packages.urllib3.exceptions.InsecureRequestWarning
         )
         py42.settings.verify_ssl_certs = False
-    password = profile.get_password()
+    password = password or profile.get_password()
     return validate_connection(profile.authority_url, profile.username, password, totp)
 
 
 def validate_connection(authority_url, username, password, totp=None):
     try:
         return py42.sdk.from_local_account(authority_url, username, password, totp=totp)
+    except SSLError as err:
+        logger.log_error(str(err))
```

Comment:
`log_error()` does already call `str()` on the err
https://github.com/code42/code42cli/blob/master/src/code42cli/logger/__init__.py#L116